File: DBService/src/main/java/com/yada/wx/db/service/model/ChinabankInfo.java
package com.yada.wx.db.service.model;

/**
 * Created by QinQiang on 2016/4/6.
 */
public class ChinabankInfo {

    private String id; //主键
    private String branchNo; //分行号
    private String name; //名称
    private String addr; //详细地址
    private String area1; //所在省
    private String area2; //所在市
    private String area3; //所在区
    private String latitude; //纬度
    private String longitude; //经度

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getBranchNo() {
        return branchNo;
    }

    public void setBranchNo(String branchNo) {
        this.branchNo = branchNo;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddr() {
        return addr;
    }

    public void setAddr(String addr) {
        this.addr = addr;
    }

    public String getArea1() {
        return area1;
    }

    public void setArea1(String area1) {
        this.area1 = area1;
    }

    public String getArea2() {
        return area2;
    }

    public void setArea2(String area2) {
        this.area2 = area2;
    }

    public String getArea3() {
        return area3;
    }

    public void setArea3(String area3) {
        this.area3 = area3;
    }

    public String getLatitude() {
        return latitude;
    }

    public void setLatitude(String latitude) {
        this.latitude = latitude;
    }

    public String getLongitude() {
        return longitude;
    }

    public void setLongitude(String longitude) {
        this.longitude = longitude;
    }
}

File: CallbackDataService/src/main/scala/com/yada/wx/cb/data/service/jpa/model/ChinaBankInfo.scala
package com.yada.wx.cb.data.service.jpa.model

import javax.persistence.{Column, Entity, Id}

@Entity(name = "T_B_CHINABANK_INFO")
class ChinaBankInfo {
  @Id
  var id: String = _
  @Column(name = "BRANCH_NO")
  var branchNo: String = _
  @Column
  var name: String = _
  @Column
  var addr: String = _
  @Column
  var area1: String = _
  @Column
  var area2: String = _
  @Column
  var area3: String = _
  @Column
  var latitude: String = _
  @Column
  var longitude: String = _
}
